CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # CHANGELOG
 
 ## Unreleased
+- Use code generation logic from Pulumi Kubernetes provider to resolve multiple bugs. [#143](https://github.com/pulumi/crd2pulumi/pull/143)
```

Comment:
Worth mentioning that this resolves a number of correctness issues, it's a big change! Also worth mentioning Patch types!

Reply:
Updated!